enforcer-rules/src/main/java/org/apache/maven/enforcer/rules/RequireSameVersions.java

```diff
@@ -69,10 +69,14 @@ public void execute() throws EnforcerRuleException {
         // consider including profile based artifacts
         Map<String, List<String>> versionMembers = new LinkedHashMap<>();
 
+        Set<String> allBuildPlugins = new HashSet<>(buildPlugins);
+        allBuildPlugins.addAll(plugins);
+        Set<String> allReportPlugins = new HashSet<>(reportPlugins);
+        allBuildPlugins.addAll(plugins);
```

Comment:
`allBuildPlugins.addAll` -> `allReportPlugins.addAll`

unit test pass, looks like not covered something

Reply:
Done.